File: src/main/java/faculdadeimpacta/com/calculoImc/models/People.java
package faculdadeimpacta.com.calculoImc.models;

public class People {
    public double Weight;
    public double Height;
    public double Imc;

    public double GetWeight(){
        return this.Weight;
    }

    public void SetWeight(double weight){
        this.Weight = weight;
    }

    public double GetHeight(){
        return this.Height;
    }

    public void SetHeight(double height){
        this.Height = height;
    }

    public double GetImc(){
        return this.Imc;
    }

    public void SetImc(double imc){
        this.Imc = imc;
    }
}

File: calculoImc/models/people.py
class People():
    def __init__(self, weight , height):
        self.weight = weight
        self.height = height
        self.imc = 0

    def from_dict(dictionary):
        weight = dictionary['weight']
        height = dictionary['height']

        people = People(weight, height)

        return people
    
    def to_dict(self):
        people = dict()
        people['weight'] = self.weight
        people['height'] = self.height
        people['imc'] = self.imc
       
        return people